finance.adoc

```diff
@@ -70,6 +70,8 @@ ADVANCE_FEE_DOWNPAYMENT::: References the netting of a cash advance fee.
 SUBSCRIPTION_CHARGE::: References a subscription charge (e.g. for the Kassaregister service).
 INVOICE_PAYMENT::: References an invoice payment.
 INVOICE_PAYMENT_FEE::: References an invoice payment fee.
+PAYMENT::: References an alternative, third-party, payment method where iZettle handles the funds.
+PAYMENT_FEE::: References the fee for a third-party payment method.
```

Comment:
The intention with these new types is to migrate all forms of payments into it. The change is kind of on going, but it will still take a while. Just thought you should keep that in mind, as you might want to reflect that here right away.